pkg/alertmanager/amcfg.go

```diff
@@ -1917,20 +1917,15 @@ func (pdc *pagerdutyConfig) sanitize(amVersion semver.Version, logger log.Logger
 }
 
 func (poc *pushoverConfig) sanitize(amVersion semver.Version, logger log.Logger) error {
-	if err := poc.HTTPConfig.sanitize(amVersion, logger); err != nil {
```

Comment:
Any special reason to remove the early return?

Reply:
Thanks for your review.  Just move to [L1944](https://github.com/prometheus-operator/prometheus-operator/pull/5886/files#diff-a66b591453b9fc72f73c1c29df8f7448add564f190f857a44c54967fde4ea6c6R1944)